sdk/src/beta9/channel.py

```diff
@@ -26,48 +25,91 @@
 from .exceptions import RunnerException
 
 
-class AuthenticatedChannel(Channel):
-    def __init__(self, *args, token: Optional[str] = None, **kwargs):
-        super().__init__(*args, **kwargs)
+class Channel(InterceptorChannel):
```

Comment:
Assuming you tested this in the runner as well? Looks good.

Reply:
Yep. Ran the code in runner locally.